modules/oxide/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/oxide",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
This seems to be an outlier. It is a minor version bump, but It isn't in versions.txt, and it doesn't have a changelog release. Is it just a special case?

Reply:
Yep, just a special case. The convention is for oxide to be a minor bump if TinyMCE is a minor bump as well